src/DDD/MASA.BuildingBlocks.DDD.Domain/Repositories/BaseRepository.cs

```diff
@@ -23,6 +23,8 @@ public virtual async Task AddRangeAsync(IEnumerable<TEntity> entities, Cancellat
 
     public abstract Task<TEntity?> FindAsync(Expression<Func<TEntity, bool>> predicate, CancellationToken cancellationToken = default);
 
+    public  abstract Task<bool> AnyAsync(Expression<Func<TEntity, bool>> predicate, CancellationToken cancellationToken = default);
```

Comment:
This is not the responsibility of IRepository

Reply:
The framework itself provides basic additions, deletions, changes, and checks, and the rest are left to the user to expand?

Reply:
processed